src/perl/lib/wtsi_clarity/clarity/process.pm

```diff
@@ -16,6 +21,126 @@ Readonly::Scalar my $OUTPUT_LIMSID       => q{./output[@output-type!="ResultFile
 Readonly::Scalar my $ALL_CONTAINERS      => q{art:details/art:artifact/location/container/@uri};
 ## use critic
 
+has '_parent' => (
+  is => 'ro',
+  isa => 'Object',
```

Comment:
Should not we add some validation relating to this attribute's type?
In the below lines we suppose it should have a 'config' and 'request' attribute...